sqlmesh/core/state_sync/common.py

```diff
@@ -206,6 +209,7 @@ def _ensure_no_gaps(
     ) -> None:
         from sqlmesh.core.scheduler import start_date
 
+        # check if w eneed to refresh here
```

Comment:
typo

Reply:
Just always fetch from state when ensuring no gaps. Bypass the cache altogether.